MPU/include/mpu.h

```diff
@@ -11,158 +11,92 @@
 #include "gpio.h"
 #include "pedals.h"
 #include "orionbms.h"
+#include "Watchdog_t4.h"
 
-class MPU
+enum
```

Comment:
What is this enum for

Reply:
Oops

Reply:
I think I implemented a better version of this is the pedals class refactor PR in the NERduino library, so I will delete